Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs

```diff
@@ -170,24 +171,22 @@ public TextDocumentHoverResponse TextDocumentHover(HoverParams hoverParams)
                     if (ScriptState == null)
                     {
                         ScriptState = await CSharpScript.RunAsync(
-                                                            code,
+                                                            string.Empty,
                                                             ScriptOptions,
                                                             cancellationToken: context.CancellationToken)
                                                         .UntilCancelled(context.CancellationToken);
                     }
-                    else
-                    {
-                        ScriptState = await ScriptState.ContinueWithAsync(
-                                                           code,
-                                                           ScriptOptions,
-                                                           catchException: e =>
-                                                           {
-                                                               exception = e;
-                                                               return true;
-                                                           },
-                                                           cancellationToken: context.CancellationToken)
-                                                       .UntilCancelled(context.CancellationToken);
-                    }
+
+                    ScriptState = await ScriptState.ContinueWithAsync(
```

Comment:
Why remove the `else` and add a run?

Reply:
And is the null check even needed if we're going to guarantee that `ScriptState` is always initialized? It looks like `GetDocumentWithOffsetFromCode` is already making that assumption.

Reply:
I removed the else because if you're executing your first cell, the `catchException` property can't be specified, so it's possible that an error could be swallowed.  This way `catchException` is always used.

`GetDocumentWithOffsetFromCode` removed the null-check and initialization; that's instead handled by the fact that the deferred commands will have caused the initialization by the time `GetDocument...` is called.

Reply:
Shhh! We were trying to hide those exceptions. 😝 

But what if there are no deferred commands?